src/lint/track_config.nim

```diff
@@ -542,6 +542,9 @@ proc checkExercisesPCP(exercises: seq[ConceptExercise] | seq[PracticeExercise];
 
   when exercises is seq[ConceptExercise]:
```

Comment:
This proc now handles Concept and Practice Exercises differently enough that a combined proc doesn't help much to reduce duplication.